app/models/ivr/connection.rb

```diff
@@ -25,4 +26,17 @@ class Ivr::Connection < ActiveRecord::Base
 
   scope :uncompleted, -> { where(status: nil, remote_id: nil) }
   scope :completed, -> { where(status: Ivr::Call::CALL_STATUSES[:completed]) }
+
+  def connecting_message_key
+    if legislator.senator?
+      'connecting_to_senator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.